### PR TITLE
Add node to trigger roslaunch_axserver based on topological node

### PR DESCRIPTION
**Pull request by hawesie**

### CODE REVIEW

**Comment by marc-hanheide** [comment]
retest this please

**Comment by marc-hanheide** [comment]
`error: package directory 'src/topological_roslaunch/triggers' does not exist`

File: topological_roslaunch/setup.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+d = generate_distutils_setup(
+    packages=['topological_roslaunch','topological_roslaunch.triggers'],
```
Comment (marc-hanheide) [inline topological_roslaunch/setup.py]: This looks problematic... does `topological_roslaunch.triggers` actually exist?

Comment (hawesie) [inline topological_roslaunch/setup.py]: Fixed it now. A dumb copy/paste.

**Pull request merged**